asset-transfer-basic/rest-api-typescript/README.md

```diff
@@ -0,0 +1,184 @@
+# Asset Transfer REST API Sample
+
+Prototype sample REST server to demonstrate good Fabric Node SDK practices
+
+The primary aim of this sample is to show how to write a long running client application using the Fabric Node SDK
+
+The REST API is intended to work with the [basic asset transfer example](https://github.com/hyperledger/fabric-samples/tree/main/asset-transfer-basic)
+
+To install the basic asset transfer chaincode on a local Fabric network, follow the [Using the Fabric test network](https://hyperledger-fabric.readthedocs.io/en/release-2.2/test_network.html) tutorial
+
+## Overview
+
+The sample creates two long lived connections to a Fabric network in order to submit and evaluate transactions using two different identities
+
+To ensure requests respond quickly enough to avoid timeouts, all submit transactions are queued for processing and will be retried if they fail
+
+Submit transactions are retried if they fail with any error, except for errors from the smart contract, or duplicate transaction errors
+
+Alternatively you might prefer to modify the sample to only retry transactions which fail with specific errors instead, for example:
+- MVCC_READ_CONFLICT
+- PHANTOM_READ_CONFLICT
+- ENDORSEMENT_POLICY_FAILURE
+- CHAINCODE_VERSION_CONFLICT
+- EXPIRED_CHAINCODE
+
+## Usage
+
+**Note:** these instructions should work with the release-2.2 branch of `fabric-samples` but later versions require some changes
```

Comment:
but this is being merged into the main branch - this doesn't make sense...

Reply:
Updated here https://github.com/hyperledgendary/fabric-rest-sample/blob/main/asset-transfer-basic/rest-api-typescript/README.md?plain=1#L30